static/app/components/calendar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import {FunctionComponent, lazy, Suspense} from 'react';
```

Comment:
You don't need to import `FunctionComponent` if you do `React.FunctionComponent` in fact, yuo can just use `React.FC`